resources/views/pages/login.twig

```diff
@@ -34,7 +34,7 @@
                     <div class="panel-body">
                         {{ msg() }}
                         {% for message in errors|default([]) %}
-                            {{ m.alert(__(message), 'danger') }}
+                            {{ m.alert(message, 'danger') }}
```

Comment:
This breaks the translation output of automatically generated validation messages in the form of `validation.[field].[type]`

Reply:
(see comment below)